account_payment_order/README.rst

```diff
@@ -0,0 +1,97 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+
+Payment order
+============================
```

Comment:
The lenght of the line should be the same as the title length. Please copy the same line above the title for highlighting the title more than the sections.